src/Microsoft.Azure.WebJobs.Protocols/FunctionDescriptor.cs

```diff
@@ -34,10 +34,14 @@ public class FunctionDescriptor
 
 #if PUBLICPROTOCOL
 #else
-        /// <summary>Gets or sets the name used for logging. This is 'Method'. </summary>
+        /// <summary>Gets or sets the name used for logging. This is 'Method' or the value overwritten by [FunctionName] </summary>
         [JsonIgnore] 
         internal string LogName { get; set; }
 
+        /// <summary>Gets or sets the name used for logging. This is ShortName or the value overwitten by [FunctionName]. </summary>
+        [JsonIgnore] 
+        internal string ShortNameOverride { get; set; }
```

Comment:
this is a pretty confusing property: 

1. should it be formatted as `Class.UserOverride`?
2. do we need two separate `ShortName` properties? I think that people will choose incorrectly